api/envoy/service/filter/v3/fcds.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
```

Comment:
Should this be named `filter_config_discovery.proto`?

Reply:
Yes. The file name seems to affect generated class names.